web/apps/labelstudio/src/components/Button/Button.styl

```diff
@@ -107,10 +107,10 @@
 
   &_type
     &_text
-      padding 0
+      --button-padding 0
```

Comment:
do we need this? 
```
&_text, &_link
      --button-padding 0
```
would handle this no?

Reply:
yep I do believe you are correct, nice catch!